austrakka/utils/helpers/project.py

```diff
@@ -0,0 +1,9 @@
+from austrakka.utils.api import api_get
+from austrakka.utils.paths import PROJECT_PATH
+
+
+def get_project_by_abbrev(abbrev: str):
+    response = api_get(
```

Comment:
You can now do this in one line with `return api_get(path=f"{PROJECT_PATH}/abbrev/{abbrev}")['data']`

`api_get` throws an `UnknownResponseException` if data is not included so there's no need to check or conditionally return the entire response as all endpoints now return the standard response